scripts/docker/debian/buster/Dockerfile.doc

```diff
@@ -0,0 +1,54 @@
+FROM debian:buster
```

Comment:
Shouldn't it be doc.Dockerfile? See https://stackoverflow.com/questions/26077543/how-to-name-dockerfiles

(Same problem also in stretch)